### ISSUE TITLE
`reportUnsafeMultipleInheritance` should not complain about `TypedDict`s

### ISSUE THREAD

**Issue by DetachHead** [DetachHead/basedpyright#345]
```py
from typing import TypedDict

class Foo(TypedDict):
    a: int

class Bar(TypedDict):
    b: str

class Baz(Foo, Bar): # error
    ...
```

**Comment by DetachHead** [comment]
i fixed this ages ago in f6e0f7348f1eeaf55ebab8f5b8796f5973ac1922
